android/src/main/res/layout/barcode_capture.xml

```diff
@@ -26,6 +26,21 @@
         android:layout_alignParentBottom="true"
         android:background="@android:color/black">
 
```

Comment:
The layout component needs some fix, the flash icon click also reverses camera
I would suggest
```
  <ImageView
            android:id="@+id/imgViewSwitchCamera"
            android:layout_width="40dp"
            android:layout_height="40dp"
            android:layout_alignParentLeft="true"
            android:layout_alignParentBottom="true"
            android:layout_marginStart="10dp"
            android:layout_marginLeft="10dp"
            android:contentDescription="Reverse camera"
            android:src="@drawable/ic_switch_camera" />
```

Reply:
What's the difference between using `ImageView` and `Button`?

btw changes pushed. Also I don't get the issue that camera gets reversed on flash icon click.